aws-inventory/lambda/inventory-client-vpn.py

```diff
@@ -0,0 +1,132 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+import os
+import time
+from datetime import datetime, timezone
+from dateutil import tz
+
+from antiope.aws_account import *
+from common import *
+
+import logging
+logger = logging.getLogger()
+logger.setLevel(getattr(logging, os.getenv('LOG_LEVEL', default='INFO')))
+logging.getLogger('botocore').setLevel(logging.WARNING)
+logging.getLogger('boto3').setLevel(logging.WARNING)
+logging.getLogger('urllib3').setLevel(logging.WARNING)
+
+RESOURCE_PATH = "ec2/vpn"
```

Comment:
Lets call this clientvpn to distinguish between site-to-site VPNs.

Reply:
updated